packages/dso-toolkit/src/components/scroll-container/scroll-container.models.ts

```diff
@@ -0,0 +1,9 @@
+export interface ScrollContainer<TemplateFnReturnType> {
+  dsoScrollContainerEvent?: (e: CustomEvent<DsoScrollContainerEvent>) => void;
+  children: TemplateFnReturnType;
+}
+
+export type DsoScrollContainerEvent = {
+  scrollEnd: "top" | "bottom" | false;
+  element: HTMLDivElement;
```

Comment:
Waarom is dit waardevol voor een afnemer van dit component?

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/fbd1c8a6-e342-416e-bb1d-f20fc379c6e8)

Het lijkt te verwijzen naar een element in het Web Component. Mis ik iets?

Reply:
Dit heeft inderdaad geen toegevoegde waarde